Update waas-delivery-optimization.md (#5109)
MicrosoftDocs/windows-itpro-docs: windows/deployment/update/waas-delivery-optimization.md

By default in Windows 10 Enterprise and Education editions, Delivery Optimization allows peer-to-peer sharing on the organization's own network only (specifically, all of the devices must be behind the same NAT), but you can configure it differently in Group Policy and mobile device management (MDM) solutions such as Microsoft Intune.

> For more details, see "Download mode" in [Delivery optimization reference](waas-delivery-optimization-reference.md).

Reviewer: If the link should take the user to the top of the page, the description text should also be more generic than "Download mode".
My suggestion for a suitable description:
```suggestion
For more details, see the [Delivery optimization reference](waas-delivery-optimization-reference.md).
```

Reviewer: @narkissit : Any thoughts on my suggestion?
